rest/data/host_create.go

```diff
@@ -390,9 +390,15 @@ func makeEC2IntentHost(taskID, userID, publicKey string, createHost apimodels.Cr
 		ec2Settings.UserData = createHost.UserdataCommand
 	}
 
+	defaultSg := evergreen.GetEnvironment().Settings().Providers.AWS.DefaultSecurityGroup
```

Comment:
Can you just grab this in the if statement so we don't get it if we don't need it?